### PR TITLE
CNV-23465: new Utilization tile info

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.13

Issue: [CNV-23465](https://issues.redhat.com//browse/CNV-23465)

Link to docs preview: https://57852--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-web-console-overview.html#ui-virtualmachine-details-overview_virt-web-console-overview

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by tmalove** [comment]
Hi @sabrinajess  Be sure to squash your commits, thanks!

**Review by lkladnit — APPROVED** [review]

File: virt/virt-web-console-overview.adoc [hunk 1]
```diff
@@ -441,7 +441,7 @@ The *Overview* tab displays resource usage, alerts, and configuration informatio
 |General VirtualMachine information
 
 |"Utilization" tile
-|*CPU*, *Memory*, *Storage*, and *Network transfer* charts
+|*CPU*, *Memory*, *Storage*, and *Network transfer* charts. By default, *Network transfer* displays the sum of all networks. To view the breakdown for a specific network, click *Breakdown by network*.
```
Comment (jeana-redhat) [inline virt/virt-web-console-overview.adoc:444]: The use of `"` for these tile names is interesting. Do you happen to have a reference for this handy? I seldom have to describe GUI items, and I would def have missed this convention if describing a tile :nerd_face:

Comment (sabrinajess) [reply virt/virt-web-console-overview.adoc:444]: Tagging @apinnick since she built this assembly and I'm not familiar with the `""` specifics.

Comment (apinnick) [reply virt/virt-web-console-overview.adoc:444]: @jeana-redhat @sabrinajess 
Our beloved [ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tile) came up with that distinction (** vs ""). Makes no sense to me but who am I to argue with Big Blue?

image

Comment (jeana-redhat) [reply virt/virt-web-console-overview.adoc:444]: Thanks, Avital! It is a little funny to me too, hopefully funny enough that I remember it when needed :sweat_smile:

**Pull request merged**